### DISCUSSION TITLE
AppleInternal Apps

### DISCUSSION THREAD

**Discussion by upintheairsheep** [elrumo/macOS_Big_Sur_icons_replacements#1457] [Icon request]
https://www.theiphonewiki.com/wiki/Apple_Internal_Apps#macOS_Internal_Apps

**Comment by OMGzilla-git** [comment]
I take it you are looking for Big Sur icons of all those apps? I can do at least some of them. I'll post back once I'm done.

**Comment by OMGzilla-git** [comment]
Ok, here are a few that I did. I'm going to pass on taking a stab at the others... I couldn't quite do justice on those. Hopefully someone else will have some good ideas for them. I've submitted what I have done... they should be live in a couple days.

AirSwitch

![AirSwitch](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/6c06bd34-b79a-4683-95cd-be2d6834f3e5)

Apple Connect

![Apple Connect](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/19dd7e03-9e04-4e09-8733-66d9ea87bfa8)

CarDisplaySim

![CarDisplaySim](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/3ec53280-9ff0-4f98-9d27-b6211b5a1b1e)

PowerWarriorX

![PowerWarriorX](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/e129e59c-a70a-42a6-8034-1301f512e361)

PurpleRabbit

![PurpleRabbit](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/6793d20c-89a2-462c-8c53-4215da69f434)

PurpleSNIFF

![PurpleSNIFF](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/ce008a93-cfd3-4859-ad4a-c7373d4f269b)

**Reply by upintheairsheep** [reply]
Tashakor!

**Comment by itjustcrashed** [comment]
I thought I'd contribute...
https://macosicons.com/#/u/wannafedora
![Apple_Chat](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/151413500/305eba50-2b05-4bc1-b398-887feff0a6f6)
